public/scss/mempool_vote_table.scss

```diff
@@ -7,67 +7,3 @@
 .matching-hash {
   background: #0078ff1c;
 }
-
-.mb-3 tbody tr td .fs14-decimal {
```

Comment:
I'm glad you are deleting these horribly brittle selectors.
But it turns out that this one in particular keeps the 'DCR' aligned to the left in the home page:
<img width="1266" alt="screen shot 2018-12-17 at 11 57 05 pm" src="https://user-images.githubusercontent.com/25571523/50139827-a418a680-0257-11e9-8f26-f86a78e374e5.png">

But rather than author any styles to do this, you could just put d-flex on the wrapper div to align things